besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -421,6 +422,8 @@ public BesuController build() {
             ethProtocolManager,
             pivotBlockSelector);
 
+    synchronizer.subscribeInSync(new PandaPrinter());
```

Comment:
IMO the right place for this is in TransitionBesuControllerBuilder rather than in the shared BesuControllerBuilder class.  It could even take the TTD as a constructor param.